gdal/doc/source/programs/gdal_translate.rst

```diff
@@ -51,6 +52,15 @@ resampling, and rescaling pixels in the process.
     Don't be forgiving of mismatches and lost data when translating to the
     output format.
 
+.. option:: -if <FORMAT>
+
+    Format/driver name to be attempted to open the input file. This is generally
+    not necessary, but can be used if automatic driver detection fails to
```

Comment:
Regarding the bug: the automatic detection could be mistaken. `-if` should have precedence over driver checks.

Reply:
> `-if` should have precedence over driver checks.

that's exactly what it does. Isn't that clear in the documentation ? If not, can you propose some wording ?

Reply:
can be used to skip the automatic driver detection

Reply:
I've just reworded it in a slightly different way, hopefully clearer